src/frontend/App/hooks/useHentOppgaverForOpprettelse.ts

```diff
@@ -0,0 +1,59 @@
+import { RessursFeilet, RessursStatus, RessursSuksess } from '../typer/ressurs';
+import { useApp } from '../context/AppContext';
+import { useCallback, useState } from 'react';
+import {
+    IOppgaverForOpprettelse,
+    OppgaveForOpprettelseType,
+} from '../../Komponenter/Behandling/Totrinnskontroll/OppgaverForOpprettelse';
+
+export interface HentOppgaverForOpprettelseState {
```

Comment:
Er dette navnet forvirrende for en type? Usikker selv, for om man skriver f.eks. `oppgaverForOpprettelse: HentOppgaverForOpprettelseState` så er det tydelig at det er en type, men kan kanskje være forvirrende? Kommer ikke på noe forslag til annet navn atm😞

Reply:
Enig at dette navnet er veldig forvirrende. Sløyfe verbet ? OppgaverForOpprettelseState ?